test/transaction.js

```diff
@@ -205,6 +205,29 @@ describe('Transaction', function () {
   })
 
   describe('hashForSignature', function () {
+    it('only uses V0 serialization', function () {
+      var randScript = new Buffer('6a', 'hex')
+
+      var tx = new Transaction()
+      tx.addInput(new Buffer('0000000000000000000000000000000000000000000000000000000000000000', 'hex'), 0)
+      tx.addOutput(randScript, 5000000000)
+      tx.___toBuffer = tx.__toBuffer
```

Comment:
I'd change it myself, but, your hot on the topic.

Could you just use `var tmp` or something that isn't almost visually identical to its predecessor?